### PR TITLE
macosx pre-req addition to the building from source/building-nym section

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/binaries/building-nym.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ sudo apt update
 sudo apt install pkg-config build-essential libssl-dev curl jq git
 ```
 
+- (Mac OS X) `pkg-config` , `brew`, `openss1`, `protobuf`, `curl`, `git`
+
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
Comment (mfahampshire) [inline src/binaries/building-nym.md:38]: does this install all of the above dependencies, or just brew?

File: src/binaries/building-nym.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ sudo apt update
...
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```
Comment (mfahampshire) [inline src/binaries/building-nym.md]: why is this here? there are download instructions for rust below

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: src/binaries/building-nym.md (hunk 1 above)
Comment (mfahampshire) [reply src/binaries/building-nym.md:38]: also if you're expecting people to run a curl script I suggest that you contextualise what it is and what it will do

Comment (ghost) [reply src/binaries/building-nym.md:38]: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" gets you all of those. curl, protobuf, pkg-config, git and openssl

Comment (mfahampshire) [reply src/binaries/building-nym.md:38]: still missing the explainer of what this script is doing - as commented above
